pom.xml

```diff
@@ -43,7 +43,7 @@
 
         <wildfly.version>16.0.0.Final</wildfly.version>
         <wildfly.build-tools.version>1.2.10.Final</wildfly.build-tools.version>
-        <eap.version>7.2.0.GA-redhat-00005</eap.version>
+        <eap.version>7.3.0.CD15-rehat-00001</eap.version>
```

Comment:
Is it a typo? Should it be redhat instead of rehat?

Reply:
fixed